apps/prairielearn/src/components/SideNav.html.ts

```diff
@@ -184,9 +184,34 @@ function CourseNav({
   const courseSideNavPageTabs = sideNavPagesTabs.course_admin;
 
   return html`
-    <div class="side-nav-section-header">Course</div>
-    <div class="side-nav-group mb-3">
-      <div class="dropdown">
+    <div class="side-nav-header">
+      <p class="header-text">Course</p>
+      <button
+        id="side-nav-toggler"
+        type="button"
+        aria-expanded="false"
+        aria-label="Toggle sidebar"
+        data-bs-toggle="tooltip"
+        data-bs-placement="right"
+        data-bs-original-title="${resLocals.side_nav_expanded
```

Comment:
👀 why is this `data-bs-original-title` and not `data-bs-title`? I don't see the former documented anywhere here: https://getbootstrap.com/docs/5.3/components/tooltips/

Reply:
Nice, I think that's a better solution!